### ISSUE TITLE
Search results page

### ISSUE THREAD

**Issue by JessicaMarine1** [GSA/smartpay-website#139]
Path:
File name: 
Design: ~~[Search Results](https://www.figma.com/file/bgNCEcglYMCDtEEBys04cx/SmartPay?type=design&node-id=1425%3A39772&t=NB2CPqpZgPZBO4xz-1)~~ 9/6/23 Decision: use search.gov standard styling
Template: ~~[Search Result Template](https://www.figma.com/file/bgNCEcglYMCDtEEBys04cx/SmartPay?type=design&node-id=1452%3A42295&t=NB2CPqpZgPZBO4xz-1)~~ 9/6/23 Decision: use search.gov standard styling
_______________ 

~~- [ ] Use search.gov with custom styling for results page~~
- [x] Search results should also include/return content contained in collapsed FAQ accordions

**Comment by mark-meyer** [comment]
This is in progress, but progress might be a little slow. We will need to obtain a subdomain like `test.smartpay.gsa.gov` before search.gov will index the site.

**Comment by mark-meyer** [comment]
The domain name request has been approved. We are now waiting on the GSA IT contractor to make the changes. Historically this has been quick.

**Comment by JessicaMarine1** [comment]
This is on hold until the code freeze is lifted.

**Comment by mark-meyer** [comment]
The search box is hooked up to search.gov now. You should be able to take it for a test ride at:

https://demo.smartpay.gsa.gov

I also added the team to the search.gov project, so you all should be able to login to https://search.gov and get to a control panel.

**Comment by rebekahperillo** [comment]
Moving to DONE. Thank you!

**Comment by mark-meyer** [comment]
Hi @rebekahperillo I am moving this back to in process to track the work getting the correct results in place (without the demo.smartpay results). I've talked to the search.gov folks and it looks like there is an entirely different search account. I have moved to this one and added you, John, Andrew, and Lora to the account.

You should no longer see the demo results when searching on staging (or in prod once we merge). There are still results from the old site (which are redirecting) — I've asked how long those will stay in the index, but have not heard back.

**Comment by rebekahperillo** [comment]
@mark-meyer Are you OK if I move this over to DONE? It looks like there are no demo results when I search in prod. Thank you!

**Comment by JessicaMarine1** [comment]
@rebekahperillo if you are satisfied with the search results in production, please feel free to move to done.

**Comment by rebekahperillo** [comment]
Moving to DONE. Look good to me, thank you.
